apps/sms/views/conversation/test/unit/information_test.js

```diff
@@ -427,7 +426,7 @@ suite('Information view', function() {
       });
     });
 
-    test('Outgoing Message report(MMS w/ subject)', function() {
+    test.skip('Outgoing Message report(MMS w/ subject)', function() {
```

Comment:
The error I get is different, I get: `RangeError: date value is not finite in DateTimeFormat.format()`

Does it ring any bell ?

note: test.skip doesn't work with our old version of mocha... it gives a javascript error and skips everything after that. And no, this is not caught by anything. Yay. :)